USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -336,6 +355,59 @@ private string RenderMarker(Marker input)
                 case FVMarker fVMarker:
                     output.AppendLine($"<span class=\"versemarker\">{fVMarker.VerseCharacter}</span>");
                     break;
+
+                case TableBlock table:
+                    output.AppendLine("<div>");
+                    output.AppendLine("<table style=\"width: 100%; \">");

```

Comment:
I don't think we should force the style on the table.  I recommend adding a class on the table that can be controlled via CSS.

Reply:
Yeah you're right. I should remain consistent with the other lines of code.